src/fe/index.js

```diff
@@ -59,12 +59,12 @@ function setColour() {
   var green = document.getElementById('green').value;
   var blue = document.getElementById('blue').value;
 
-  var colour = [red, green, blue];
+  // var colour = [red, green, blue];
 
   // post the /api/vi/set_colour endpoint
-  fetch('/api/v1/set_colour', {
+  fetch(`/api/v1/set_colour?red=${red}&green=${green}&blue=${blue}`, {
```

Comment:
nice - this is using "query parameters" instead of the POST "body" to send the colour information

That's allgood - but it's a deviation from the spec - mind updating the app doc spec to mention query parameters for this end point also? (like you did for the brightness one)